pkg/controller/storagecluster/topology_test.go

```diff
@@ -246,7 +271,16 @@ func TestFailureDomain(t *testing.T) {
 			expectedFailureDomain: "rack",
 		},
 		{
-			label: "Case 3", // storagecluster with three or more zone topology labels
+			label: "Case 3", // storagecluster has predefined failure domain of `host`
+			storageCluster: &api.StorageCluster{
+				Status: api.StorageClusterStatus{
+					FailureDomain: "host",
+				},
+			},
+			expectedFailureDomain: "host",
+		},
+		{
+			label: "Case 4", // storagecluster with three or more zone topology labels
```

Comment:
Add this as case 5. I see what you're going for, but here I would value the cleaner patch and chronological order over other logical grouping of test cases.